be/src/exec/vectorized/aggregate/agg_hash_map.h

```diff
@@ -80,9 +106,13 @@ struct AggHashMapWithOneNumberKey {
         DCHECK(!key_columns[0]->is_nullable());
         auto column = down_cast<ColumnType*>(key_columns[0].get());
 
-        for (size_t i = 0; i < column->size(); i++) {
+        PRECOMPUTE_HASH_VALUES_NUMBER(column, 16);
```

Comment:
why 16 and why this prefetch will help improve performance?

Reply:
There is no specific reason why 16 works and 16 works the best. But we can pretty be sure that prefetch has a positive effect.

Because after doing the profile, I find that there are a lot of cache miss events when loading hash table(slot array and control array). And prefetch can help.

Reply:
better to 16 a `static const xxx`, add the comment above, use that variable instead.

Reply:
I think it's just an empirical value and very hard to explain why to use this value. And macro is

```
#define PRECOMPUTE_HASH_VALUES_NUMBER(column, prefetch_dist)    
```

It's pretty self-explained.